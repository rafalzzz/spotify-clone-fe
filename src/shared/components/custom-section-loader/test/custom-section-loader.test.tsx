import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import { CustomSectionLoader } from '../';

describe('CustomSectionLoader', () => {
  it('renders component without error', () => {
    const screen = render(<CustomSectionLoader />);
    expect(screen).toMatchSnapshot();
  });
});
