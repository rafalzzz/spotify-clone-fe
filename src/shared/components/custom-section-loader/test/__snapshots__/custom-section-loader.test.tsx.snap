// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomSectionLoader renders component without error 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="custom-section-loader"
      >
        <div
          class="custom-section-loader__header"
        >
          <div
            class="custom-section-loader__header__title"
          />
          <div
            class="custom-section-loader__header__redirection"
          />
        </div>
        <ul
          class="custom-section-loader__items"
        >
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
          <li>
            <div
              class="custom-section-item-loader"
            >
              <div
                class="custom-section-item-loader__image"
              />
              <div
                class="custom-section-item-loader__skeleton-text__container"
              >
                <div
                  class="custom-section-item-loader__skeleton-text__header"
                />
                <div
                  class="custom-section-item-loader__skeleton-text__subheader"
                />
              </div>
            </div>
          </li>
        </ul>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="custom-section-loader"
    >
      <div
        class="custom-section-loader__header"
      >
        <div
          class="custom-section-loader__header__title"
        />
        <div
          class="custom-section-loader__header__redirection"
        />
      </div>
      <ul
        class="custom-section-loader__items"
      >
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
        <li>
          <div
            class="custom-section-item-loader"
          >
            <div
              class="custom-section-item-loader__image"
            />
            <div
              class="custom-section-item-loader__skeleton-text__container"
            >
              <div
                class="custom-section-item-loader__skeleton-text__header"
              />
              <div
                class="custom-section-item-loader__skeleton-text__subheader"
              />
            </div>
          </div>
        </li>
      </ul>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
